components/calendar/calendar.js

```diff
@@ -29,15 +29,13 @@ const keyCodes = {
 const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
 
 let calendarData;
-function getCalendarData(refresh) {
-	if (!calendarData) {
-		calendarData = {};
-		calendarData.descriptor = getDateTimeDescriptorShared(refresh);
-		calendarData.firstDayOfWeek = calendarData.descriptor.calendar.firstDayOfWeek;
-		calendarData.daysOfWeekIndex = [];
-		for (let i = calendarData.firstDayOfWeek; i < calendarData.firstDayOfWeek + daysInWeek; i++) {
-			calendarData.daysOfWeekIndex.push(i % daysInWeek);
-		}
+function getCalendarData() {
+	calendarData = {};
+	calendarData.descriptor = getDateTimeDescriptor();
```

Comment:
The culprit was actually two separate places where we were saving local caches -- `calendarData` here and in `getDateTimeDescriptorShared`.

Previously the caches were invalidated in calendar's event handler for `d2l-localize-resources-change`, but because of the isolated fixtures that event fires before any calendars are even put on the page.

I'm going to separately look into adding a cache inside the `@brighspace-ui/intl` library, since it's in a much better place to invalidate it.